src/QDMEwald/src/QDMEwald.hpp

```diff
@@ -0,0 +1,204 @@
+/*
+ *            Copyright 2019 Juelich Supercomputing Centre
```

Comment:
@rhalver is that what you need?

Reply:
Copyright 2019 Forschungszentrum Juelich GmbH, Germany

It needs to mention the research center itself, not the institute, usually we mention the authors themselves  in it, but I am not sure if it is required.